.travis.yml

```diff
@@ -4,22 +4,24 @@ env:
     - DATABASE_URL='postgres://postgres@localhost/user_management'
     - TERM='xterm-256color'
   matrix:
-    - TOX_ENV=py27-django111
-    - TOX_ENV=py34-django111
     - TOX_ENV=py35-django111
-    - TOX_ENV=py27-djangopre
-    - TOX_ENV=py34-djangopre
+    - TOX_ENV=py36-django111
     - TOX_ENV=py35-djangopre
+    - TOX_ENV=py36-djangopre
```

Comment:
It's probably worthwhile adding 3.7 and 3.8 now. I'd be happy to drop 3.5 too, since that's security fixes only now.

Reply:
3.5 will be end of life in September: https://devguide.python.org/#status-of-python-branches